browser/ethereum_remote_client/ethereum_remote_client_service.cc

```diff
@@ -399,3 +399,9 @@ void EthereumRemoteClientService::MaybeLoadCryptoWalletsExtension(
   ethereum_remote_client_delegate_->MaybeLoadCryptoWalletsExtension(context_);
 #endif
 }
+
+void EthereumRemoteClientService::RemoveCryptoWalletExtension() {
+#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
`#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED) && BUILDFLAG(ENABLE_EXTENSIONS)` from 
`#include "brave/browser/ethereum_remote_client/buildflags/buildflags.h"`

Reply:
its declaration and definition are already in `#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)` guard